src/workloads/execution/Densify.yml

```diff
@@ -0,0 +1,194 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload test tests the performance of the $densify stage. $densify can operate on both dates
+  and numbers.
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000
+    BatchSize: &batchSize 1000
+    Document:
+      partitionKey: {^RandomInt: {min: 1, max: 100}}
```

Comment:
Do we think it would be interesting to test differences in number of partitions? I don't think it should have an impact, except maybe in cases where we need to finish densifying after the input.